tests/phpunit/ExtractorTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\FtpExtractor\Tests;
+
+use Keboola\FtpExtractor\ConfigDefinition;
+use Keboola\FtpExtractor\FtpExtractorComponent;
+use Keboola\Temp\Temp;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\NullLogger;
+use Symfony\Component\Filesystem\Filesystem;
+
+class ExtractorTest extends TestCase
+{
+    public function testConnection(): void
+    {
+        $temp = new Temp('ex-storage');
+        $temp->initRunFolder();
+        $fs = new Filesystem();
+
+        $configFile = [
+            'action' => 'run',
+            'parameters' => [
+                'host' => 'ftp',
+                'username' => 'ftpuser',
+                'password' => 'userpass',
+                'port' => 21,
+                'path' => 'dir1/*',
+                'wildcard' => true,
+                'connectionType' => ConfigDefinition::CONNECTION_TYPE_FTP,
+            ],
+        ];
+
+        $baseDir = $temp->getTmpFolder();
+        $fs->dumpFile($baseDir . '/config.json', json_encode($configFile));
+        putenv('KBC_DATADIR=' . $baseDir);
+        $fs->mkdir($baseDir . '/out/files/');
+
+        $app = new FtpExtractorComponent(new NullLogger());
+        $app->run();
```

Comment:
Test by měl něco testovat. Tady netestuje nic, tedy kromě toho, že se to dá spustit, což IMHO dostatečně testují datadir testy

Reply:
neni tam stary phpunit? novy phpunit by mel zarvat, ze test nema assertions

Reply:
Řve to, že tam nejsou assertions a že je ten test risky. Ještě jsem se tak neskamarádil s datadir-testa (debugovat je) a tak jsem si tam udělal tenhle test. Po dokončení vývoje ho smažu. Mám ho jako místo, kde se ta komponenta dá jednoduše spustit.

Reply:
jasný. Chápu:) odstranil jsem test a budu spouště jak popsal @odinuv